frontend/src/metabase/search/components/SearchFilterModal/SearchFilterModal/SearchFilterModal.tsx

```diff
@@ -64,11 +84,10 @@ export const SearchFilterModal = ({
     setIsOpen(false);
   };
 
-  // we can use this field to control which filters are available
-  // - we can enable the 'verified' filter here
-  const availableFilters: FilterTypeKeys[] = useMemo(() => {
-    return [SearchFilterKeys.Type];
-  }, []);
+  const availableFilters: FilterTypeKeys[] = useMemo(
+    () => [SearchFilterKeys.Type, SearchFilterKeys.Verified],
+    [],
+  );
```

Comment:
Should we lift this up outside the component as a constant as it doesn't rely on any props?

Reply:
There's some cleanup that I'm working on with the `created_by` and `created_at`, I'm going to put that as a separate PR so I could separate the logic of that from the general `Verified` filter stuff.